podcasts/Onboarding/Import/ImportLandingView.swift

```diff
@@ -5,11 +5,6 @@ struct ImportLandingView: View {
     let viewModel: ImportViewModel
 
     var body: some View {
-        // It's possible the user has no supported apps installed
```

Comment:
No need to check that anymore. We will always have at least 2 available resources: others and opml from url